sdk/attestation/azure-security-attestation/src/test/java/com/azure/security/attestation/AttestationClientTestBase.java

```diff
@@ -135,209 +175,181 @@ ClientTypes classifyClient(String clientUri) {
+            : Configuration.getGlobalConfiguration().get("isolatedSigningCertificate");
     }
 
-    protected X509Certificate getIsolatedSigningCertificate() {
-        return X509CertUtils.parse(Base64.getDecoder().decode(getIsolatedSigningCertificateBase64()));
+    protected static X509Certificate getIsolatedSigningCertificate() {
+        String base64Certificate = getIsolatedSigningCertificateBase64();
+        return X509CertUtils.parse(Base64.getDecoder().decode(base64Certificate));
     }
 
     /**
      * Retrieve the signing key used for the isolated attestation instance.
+     *
      * @return Returns a base64 encoded RSA Key used to sign policy documents.
      */
-    String getIsolatedSigningKeyBase64() {
-        String signingKey = Configuration.getGlobalConfiguration().get("isolatedSigningKey");
-        if (signingKey != null) {
-            return signingKey;
-        }
-
-        // Use a pre-canned signing key captured at provisioning time.
-        return "MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQC9DOaEg982MKbxAX6JwO7MN3w4IDa1v9kRb9s66gqDBjbYM363wfZ"
-            + "Pmf6LAMt6+8IwnIhpLB7Y1F6wPUN/PdoOIaFqUpm3koXOnn0UNbRBGatO0xOu4k6oa2XWHzapsq2jG81GOm/JGdTxpp304ML4cswlJJ4"
-            + "HEjNuL2cqz/U6Ti4WnUJLwqyxRjyCX7gsr/J8hc/VRHEGfIPqghO6Hc6RnkvI0W9gTRUKxqyktLLLW02mSqFKr2zaDpO5qM3lNNo3KQK"
-            + "M1pJIHrF5rZYOQbFxE4HG/NiE6V5Ttlm9mw7McKC2tlTW7D241tvZh+iCYsLQVHZRC5Ckv7A9jVKNr96FAgMBAAECggEANHxnZ3UBPJd"
-            + "HUfP7QJ167uGcsCxpZA5OYqPudq3Rm0zXXzD0xmmoS7NSUx82Xv+Ui1B6ItDoA4T6gO2RVrtCVdgKa21hVfDtC08/JKRAVi6dO/YoKc6"
-            + "Uzfd7P3ZNA9IimP04AX5p2Kg+UZTTl8/q8XVVrh47bXUlKDsq5lz0FWpyWY8QuwkSZEg45kdNK6ebYUlSAdJNzhPT1NyUwm8rVRXDYdE"
-            + "tEH+Dpo3WPwv5PmHARNM5d6MtZJCFOAdXrSlhIJmHW9yrDb+93UIEZADWEXzDo1VgRMBlG0U8GAsIWcTrBETwSBpTPG6rcyTFd5EP0Bc"
-            + "82k4C8J+R6rgsZcoixQKBgQD0KU6zk0bDAyaQ24k5/sxL2yRyYsB7skJj6ICsGePgdiS54uHAI+ZlWahd6sc6xzYcrdyVoiTiGJGZBjk"
-            + "xwAl/C/m37z+qSOtOPbOv7Vwm7neeh5pwAFWS5TSTP5eYoh3Jy58C8aC7DdQEXFym14rFvn0EVeFLjbQ8GX+yHElBFwKBgQDGN4TrTGq"
-            + "r2jzrSEk2MuMdol6TOFi2h7avrc5WNQ7A0CTSKS6bOB2iQRZ5NGNXbO/tjc1O25AymRvtyot1TjI0POOuUqANg+G++8NlwV+jRLt4/bh"
-            + "vxq6V5poYz4vVqOt8XMoud6P2ZUEdyMbm8Mb5CyhgIgq40brP4svVrwFGwwKBgQDbZ7M9U38nBtHFIkJyRzE2iTp7P5pQjaNzPB9EiA9"
-            + "qSUA/ek3650fGk4bkvnFBtHo/00yBei9CrWlI5XrH5hWigRRp4SiI0PYCXRf3Y4iHw4rifmCdfO9pDWbtWxvTIPs7tyqw7ojNDmQTdxw"
-            + "bIV2mzMJy/ulxh2kLCK7jUt/RCQKBgF2GEJcc5S/Utu2km7aQ6AwxckIwUds9yXq2nSabvixHiTgH6k+eeIz3/OyPRVgYtFP3lilg+3a"
-            + "EO2NFL+vXS2TOXfcAUThPQ5JjI9y9EqzsXzBFNmK28eUUexy9wJK9Mu7osRSfqqM1PzYYZ5DLhnY3R4OkTD7L5kL0FYzrF8UVAoGBAKy"
-            + "hZZuD4sMS3WsDg9d+LfhnGcKTsBfovi7KSqHLLZmqJ6sxKXGoWe9IlGadfWk3VNIW/LQ+/hipZv4bsOCBcOc097+Ev23u6bl2Xm0LHq9"
-            + "FFUDqTfD8qTpQF+GWOEXfdV5NfwmwRc1MLcAa3BFwvRgG+yCoMZgP4PGE3rhdakWF";
+    static String getIsolatedSigningKeyBase64() {
+        return TEST_MODE == TestMode.PLAYBACK
```

Comment:
why aren't we using `interceptorManager` directly?

Reply:
These are static methods that get used by ParameterizedTests, so they can't take method arguments.